src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -16,6 +16,8 @@
 #include "Calculator/Views/GraphingCalculator/EquationInputArea.xaml.h"
 #include "CalcViewModel/Common/Utils.h"
 #include "GraphingSettings.xaml.h"
+#include <iomanip>
```

Comment:
nit: We should put this and the below header file in pch.h

Reply:
done, is there a "best-practices" for what to include/not include in the pch.h files?

Reply:
The style of the project is if its a standard header library it goes into the pch.h, otherwise can go in the regular .h

Reply:
awesome, thanks.